static/js/manage/backlog-overview.js

```diff
@@ -6,6 +6,10 @@ MEDUSA.manage.backlogOverview = function() {
         }
     });
 
+    $.ajaxEpSearch({
```

Comment:
@fernandog we can add another option to the ajaxEpSearch to remove the row if found or even replace the whole row with a new one if we want.

We can also base the archive button's code form what this uses.